demo/lib/numajson2qemuopts.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
+
+"""numajson2qemuopts - convert NUMA node list from JSON to Qemu options
```

Comment:
Could we also lower/eliminate the bar for including the emulation of any target hardware into the test set ? This could be as simple as adding a simple helper script `numactlH2numajson` which would take as input output from `numactl -H` and convert it to the format accepted by this script.

Reply:
Sounds good!